ctf-gamemode/ctf_client.lua

```diff
@@ -79,7 +81,7 @@ AddEventHandler("SetObjectiveVisible", function(flagEntityNetID, bVisible)
 
         print("SetObjectiveVisible: " .. GetEntityArchetypeName(flagEntity) .. " to our player, owner is: " .. GetPlayerName(NetworkGetEntityOwner(flagEntity)))
 
-        SetEntityVisible(flagEntity, bVisible, 0)
```

Comment:
Last parameter is `bResetRenderPhaseVisibilityMask` a boolean parameter, by default it's always `false`.

Seems to only be used to reset the visibility masks (if they were modified by other methods), which basically assigns all visibility masks again via ORing.

Reply:
You should make a PR to the natives repo since this parameter is not documented. I removed it because it was not a bool and there was no reason to use it.

Reply:
> You should make a PR to the natives repo since this parameter is not documented. I removed it because it was not a bool and there was no reason to use it.

Yes, I will probably update the natives repo soon with the findings I had.